pkg/ovs/ovsconfig/default.go

```diff
@@ -1,12 +1,33 @@
 // +build !windows
+// Copyright 2020 Antrea Authors
```

Comment:
Add a line between the "build" tag and license code.

Reply:
Done